src/main/java/com/marginallyclever/makelangelo/plotter/plotterControls/SaveGCode.java

```diff
@@ -56,17 +59,38 @@ private String addExtension(String name, String [] extensions) {
 		return name + "." + extensions[0];
 	}
 	
+	boolean detailGCodeHeader = true;
+	
 	private void save(String filename, Turtle turtle, Plotter robot) throws Exception {
 		logger.debug("saving...");
 		
 		try (Writer out = new OutputStreamWriter(new FileOutputStream(filename))) {
+			
+			if (detailGCodeHeader) {
```

Comment:
when this boolean is set to `false` ?

Reply:
I have the good or bad habit of making blocks subject to a decarred boolean as a class attribute ... easier to deactivate when recoding / less clean for a production version ...